README.md

```diff
@@ -42,6 +42,7 @@ You may want to see how the model is defined or how parallelism techniques are a
 10. DDP and HSDP
 11. All options easily configured via [toml files](train_configs/)
 12. [Interoperable checkpoints](docs/checkpoint.md) which can be loaded directly into [`torchtune`](https://github.com/pytorch/torchtune) for fine-tuning
+13. [Memory profier](docs/memory_profiler.md) dump memory snapshots
```

Comment:
maybe modify ine 38?  "6. Loss, GPU memory, tokens-per-second, and MFU displayed and logged via [TensorBoard](https://github.com/pytorch/torchtitan#tensorboard)" ? 

this is a debugging feature instead of a major PTD feature

Reply:
We should add an item in the Debugging section together with Flight Recorder. Personally I don't mind creating a new entry in "key features" about debugging tools, but they should be both included or both excluded here.

Reply:
Combine them together in line 45 as bullet point 13.